parser/src/main/java/com/distkv/parser/DistkvNewSqlListener.java

```diff
@@ -32,165 +30,206 @@ public DistkvParsedResult getParsedResult() {
   public void enterExit(DistkvNewSQLParser.ExitContext ctx) {
     Preconditions.checkState(parsedResult == null);
     Preconditions.checkState(ctx.children.size() == 1);
-    parsedResult = new DistkvParsedResult(RequestTypeEnum.EXIT, null);
+    parsedResult = new DistkvParsedResult(RequestType.SET_EXISTS, null);
```

Comment:
`EXIT` is not `exists`.

It means we will exit the command line tool with `system.exit(0);`